src/champion-decisions.md

```diff
@@ -0,0 +1,41 @@
+# Champion decisions
+
+Champion decisions do not represent team consensus. Rather, they indicate that *somebody* on the team is willing to champion an idea. We use them to begin experiments and for other decisions where we want to move quickly and iterate.
+
+## When to use champion decisions
+
+* Starting a [lang-team experiment](./how_to/experiment.md)
+* Closing an RFC or issue that is clearly not going to be accepted
+* Other lightweight decisions that don't make a durable commitment
+
```

Comment:
```suggestion

Note that many decisions made over the course of an experiment do not require a formal champion decision. Here are examples of decisions that do **not** require nomination:

* Removing or sunsetting an experiment for reasons outside the lang team's direct control
* Cutting scope or focusing on a particular sub-problem
* Pursuing a particular direction in line with the original experiment's goals
* New constraints you've learned as part of exploring the design space

Note that all of these should still appear in a champion's regular updates to the team so that concerns can be surfaced.
```